### PR TITLE
FW-3046-QR-code-WOTD-widget

### PR DESCRIPTION
**Pull request by Cara-Barter**
### Description of Changes

added QR button component to WOTD widget, adjusted styling

### Checklist

- [ ] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable

### Reviewers Should Do The Following:

- [x] Review the changes here
- [ ] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: src/components/ShareLinks/ShareLinksPresentation.js [hunk 1]
```diff
@@ -5,7 +5,16 @@ import copyToClipboard from 'common/utils/copyToClipboard'
...
-function ShareLinksPresentation({ url, title, modalCloseHandler }) {
+import { QrcodeButton } from 'components/Actions'
+import { makePlural } from 'common/utils/urlHelpers'
+
+function ShareLinksPresentation({
+  url,
+  sitename,
+  title,
+  modalCloseHandler,
+  entry,
```
Comment (gmcauliffe) [inline src/components/ShareLinks/ShareLinksPresentation.js:16]: I see that you've added some new props to ShareLinksPresentation (entry and sitename). You will need to pass these in wherever ShareLinksPresentation is being used.

**Review by gmcauliffe — CHANGES_REQUESTED** [review]
This is looking good. Just one issue regarding the new props that ShareLinks will now need. Let me know if you want clarification 👍

File: src/components/ShareLinks/ShareLinksPresentation.js (hunk 1 above)
Comment (Cara-Barter) [reply src/components/ShareLinks/ShareLinksPresentation.js:16]: I *think* its all fixed now :)

**Review by gmcauliffe — APPROVED** [review]
LGTM 👍

**Pull request merged**